From c8638fe90efdbd0bb88432b884a9a63ba67f1b12 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Sun, 27 Oct 2024 13:25:32 +0000
Subject: [PATCH 1/1] packagekit: HACK! Disable req. for repo plugin to show n
 apps installed

Solus currently doesn't have a repo management plugin for G-S. This
works, but I imagine it'll be a lot quicker to resolve if we did.
---
 plugins/packagekit/gs-plugin-packagekit.c | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/plugins/packagekit/gs-plugin-packagekit.c b/plugins/packagekit/gs-plugin-packagekit.c
index bfe236187..24093c6de 100644
--- a/plugins/packagekit/gs-plugin-packagekit.c
+++ b/plugins/packagekit/gs-plugin-packagekit.c
@@ -2324,17 +2324,16 @@ gs_plugin_packagekit_refine_async (GsPlugin            *plugin,
 		if (gs_app_has_quirk (app, GS_APP_QUIRK_IS_WILDCARD))
 			continue;
 
-		if (!gs_app_has_management_plugin (app, NULL) &&
+		/*if (!gs_app_has_management_plugin (app, NULL) &&
 		    !gs_app_has_management_plugin (app, GS_PLUGIN (self)))
-			continue;
+			continue;*/
 
 		n_considered++;
 
 		/* Repositories */
 		filename = gs_app_get_metadata_item (app, "repos::repo-filename");
 
-		if (gs_app_get_kind (app) == AS_COMPONENT_KIND_REPOSITORY &&
-		    filename != NULL) {
+		if (gs_app_get_kind (app) == AS_COMPONENT_KIND_REPOSITORY) {
 			gs_app_list_add (repos_list, app);
 		}
 
@@ -2778,9 +2777,9 @@ sources_related_got_installed_cb (GObject      *source_object,
 		    gs_app_get_id (app) == NULL)
 			continue;
 
-		if (!gs_app_has_management_plugin (app, NULL) &&
+		/*if (!gs_app_has_management_plugin (app, NULL) &&
 		    !gs_app_has_management_plugin (app, GS_PLUGIN (self)))
-			continue;
+			continue;*/
 
 		g_hash_table_insert (sources_hash, g_strdup (gs_app_get_id (app)), app);
 	}
-- 
2.47.0

